api.planx.uk/modules/flows/validate/service/projectTypes.ts

```diff
@@ -0,0 +1,69 @@
+import { getValidSchemaValues } from "@opensystemslab/planx-core";
+import { ComponentType, FlowGraph } from "@opensystemslab/planx-core/types";
+import countBy from "lodash/countBy";
+
+import { isComponentType } from "../helpers";
+import { FlowValidationResponse } from "./index";
+
+const validateProjectTypes = (flowGraph: FlowGraph): FlowValidationResponse => {
+  // Get all passport values set by Answers of Checklists that set fn "proposal.projectType"
+  const projectTypeVals = getProjectTypeVals(flowGraph);
+  if (projectTypeVals.length < 1) {
+    return {
+      title: "Project types",
+      status: "Not applicable",
+      message:
+        'Your flow is not using Checklists which set "proposal.projectType"',
+    };
+  }
+
+  // Get all project types supported by current release of ODP Schema & compare
+  const validProjectTypes = getValidSchemaValues("ProjectType");
+  const invalidProjectVals: string[] = [];
+  projectTypeVals.forEach((val) => {
+    if (!validProjectTypes?.includes(val)) {
+      invalidProjectVals.push(val);
+    }
+  });
```

Comment:
nit: A `filter()` might be a bit more semantically meaningful here. I think we do use `difference()` from lodash for similar checks elsewhere.